docs/user-guide/grpc.md

```diff
@@ -8,102 +8,97 @@ Ambassador makes it easy to access your services from outside your application.
 
-service Greeter { ... }
+service Greeter1 { ... }
 ```
 
-so its URL prefix is `helloworld.Greeter`, and a reasonable `Mapping` would be:
+so its URL prefix is `helloworld.Greeter1`, and a reasonable `Mapping` would be:
 
 ```yaml
 ---
 apiVersion: ambassador/v0
 kind: Mapping
 name: grpc_mapping
 grpc: true
-prefix: /helloworld.Greeter/
-rewrite: /helloworld.Greeter/
-service: grpc-greet
+prefix: /helloworld.Greeter1/
+rewrite: /helloworld.Greeter1/
+service: grpc-example
 ```
 
 Note the `grpc: true` line -- this is the necessary magic when mapping a gRPC service. Also note that you'll need `prefix` and `rewrite` the same here, since the gRPC service needs the package and service to be in the request to do the right thing.
 
-## Deploying `Hello World`
+## Deploying `gRPC Example`
 
-To deploy and map `Hello World`, we can use the following YAML:
+To deploy and map `gRPC Example`, we can use the following YAML:
 
 ```yaml
 ---
 apiVersion: v1
 kind: Service
 metadata:
   labels:
-    service: grpc-greet
-  name: grpc-greet
+    service: grpc-example
+  name: grpc-example
   annotations:
     getambassador.io/config: |
       ---
       apiVersion: ambassador/v0
       kind: Mapping
       name: grpc_mapping
       grpc: true
-      prefix: /helloworld.Greeter/
-      rewrite: /helloworld.Greeter/
-      service: grpc-greet
+      prefix: /helloworld.Greeter1/
+      rewrite: /helloworld.Greeter1/
+      service: grpc-example
 spec:
   type: ClusterIP
   ports:
   - port: 80
-    name: grpc-greet
+    name: grpc-example
     targetPort: grpc-api
   selector:
-    service: grpc-greet
+    service: grpc-example
 ---
 apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
-  name: grpc-greet
+  name: grpc-example
 spec:
   replicas: 1
   template:
     metadata:
       labels:
-        service: grpc-greet
+        service: grpc-example
     spec:
       containers:
       - name: grpc-greet
-        image: enm10k/grpc-hello-world
+        image: nkrause/grpc_example:latest
         ports:
         - name: grpc-api
-          containerPort: 9999
-        env:
-          - name: PORT
-            value: "9999"
-        command:
-          - greeter_server
+          containerPort: 50052
       restartPolicy: Always
 ```
 
-(We tell the gRPC service to run on port 9999, then map the container's port 80 inbound to simplify the `Mapping`. There's no magic behind these port numbers: anything will work as long as you're consistent in when mapping everything.)
+(We tell the gRPC service to run on port 50052, then map the container's port 80 inbound to simplify the `Mapping`. There's no magic behind these port numbers: anything will work as long as you're consistent in when mapping everything.)
 
 This is available from getambassador.io, so you can simply
 
 ```shell
-kubectl apply -f https://getambassador.io/yaml/demo/demo-grpc.yaml
+curl https://raw.githubusercontent.com/datawire/grpc-example/master/grpc_example.yaml | kubectl apply -f -
```

Comment:
I'd rather stick with the status quo gRPC infrastructure, because the demo-grpc.yaml is in the Ambassador tree, and adding third party dependencies means it's less likely to be tested.